modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="installation-about-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VPC in Google Cloud Platform (GCP). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing GCP VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself.
+
+[id="installation-about-custom-gcp-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+The installation program will no longer create the following components:
+* VPC
+* Subnets
+* Cloud Router
+* Cloud NAT
+* NAT IP addresses
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC and subnets must meet the following characteristics:
+
+* The VPC must be in the same GCP project that you deploy the {product-title} cluster to.
+* To allow access to the internet from the control plane and compute machines, you must configure Cloud NAT on the subnets to allow egress to it. These machines do not have a public address. Even if you do not require access to the internet, you must allow egress to the VPC network to obtain the installation program and images. Because multiple Cloud NATs cannot be configured on the shared subnets, the installation program cannot configure it.
+
+To ensure that the subnets that you provide to the installation program are suitable, it confirms the following data:
+
+* All the subnets that you specify exist and belong to the VPC that you specified.
+* The subnet CIDRs belong to the machine CIDR.
+* You must provide a subnet to deploy the cluster control plane and compute machines to. You can use the same subnet for both machine types.
```

Comment:
I get what you're saying here, but it's awkward to read this bullet: "the program confirms the data that you must provide a subnet. A couple of suggestions:
* Remove the word "data" from the intro sentence
* Change the last bullet to something like, "You are providing a subnet to deploy the cluster..."

I get that you are emphasizing that they "must provide", so if that's the most important thing, then the bullet's probably fine as you have it.

Reply:
I blame IBM style.

> :no_entry_sign: following n
Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”

> Write in the simple present tense as much as possible.

Reply:
Hadn't noticed that before. That's annoying.